src/main/java/io/github/cottonmc/cotton/gui/widget/WTextField.java

```diff
@@ -312,51 +385,60 @@ public boolean mouseClicked(double double_1, double double_2, int int_1) {
 	@Environment(EnvType.CLIENT)
 	public void renderButton(int x, int y) {
 		if (this.font==null) this.font = MinecraftClient.getInstance().textRenderer;
-		
-		int borderColor = (this.isFocused()) ? 0xFF_FFFFA0 : 0xFF_A0A0A0;
-		ScreenDrawing.coloredRect(x-1, y-1, width+2, height+2, borderColor);
-		ScreenDrawing.coloredRect(x, y, width, height, 0xFF000000);
-		
+
+		int borderColor = (this.isFocused()) ? focusedColor : notFocusedColor;
+		if (drawFocusBorder) ScreenDrawing.coloredRect(x-1, y-1, width+2, height+2, borderColor);
+		ScreenDrawing.coloredRect(x, y, width, height, backgroundColor);
+
 
 		int textColor = this.editable ? this.enabledColor : this.uneditableColor;
-		
+
 		//TODO: Scroll offset
-		String trimText = font.trimToWidth(this.text, this.width-OFFSET_X_TEXT);
-		
+		String trimText = font.trimToWidth(this.getText(), this.width-OFFSET_X_TEXT);
+
 		boolean selection = (select!=-1);
 		boolean focused = this.isFocused(); //this.isFocused() && this.focusedTicks / 6 % 2 == 0 && boolean_1; //Blinks the cursor
-		
+
 		//int textWidth = font.getStringWidth(trimText);
 		//int textAnchor = (font.isRightToLeft()) ?
 		//		x + OFFSET_X_TEXT + textWidth :
 		//		x + OFFSET_X_TEXT;
-		
+
 		int textX = x + OFFSET_X_TEXT;
 				//(font.isRightToLeft()) ?
 				//textAnchor - textWidth :
 				//textAnchor;
-		
+
 		int textY = y + (height - 8) / 2;
-		
+
 		//TODO: Adjust by scroll offset
 		int adjustedCursor = this.cursor;
 		if (adjustedCursor > trimText.length()) {
 			adjustedCursor = trimText.length();
 		}
-		
+
 		int preCursorAdvance = textX;
 		if (!trimText.isEmpty()) {
 			String string_2 = trimText.substring(0,adjustedCursor);
-			preCursorAdvance = font.drawWithShadow(string_2, textX, textY, textColor);
+			if (drawTextWithShadow)
+				preCursorAdvance = font.drawWithShadow(string_2, textX, textY, textColor);
+			else
+				preCursorAdvance = font.draw(string_2, textX, textY, textColor);
```

Comment:
`if`s should use braces.

Reply:
Would you prefer the same for inline if's ?

Reply:
No, inline ifs can be without.